Cannot run map slicer via command line

Here's a test folder containing a 1.20.1 icons.png and 1.20.4 map_icons.png:
[simpleassetstest.zip](https://github.com/Mojang/slicer/files/14694303/simpleassetstest.zip)

With the 1.20.2-1.1.3 gui splitter jar file and the 1.20.5-1.1.4 map icon splitter jar file in the same directory as the assets, running a command via Windows PowerShell (I may test cygwin later) will correctly allow the 1.20.2 slicer to cut up the icons.png file, but will return an error when trying to use 1.20.5's slicer.

Command used: java -jar [name of jar].jar [directory containing the assets folder] [directory containing the assets folder]/output [directory containing the assets folder]/remainder

When using the 1.20.2 splitter, all works fine. 1.20.5's splitter, however, gives this error:

Error: Could not find or load main class com.mojang.slicer.Main
Caused by: java.lang.ClassNotFoundException: com.mojang.slicer.Main

I have no idea if I have something configured wrong behind the scenes, but the fact that one jar runs and the other faceplants is concerning.

--- a/1.20.5/src/main/java/com/mojang/slicer/Main.java
+++ b/1.20.5/src/main/java/com/mojang/slicer/Main.java
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-package slicer;
+package com.mojang.slicer;
 
 import com.mojang.slicer.Box;
 import com.mojang.slicer.InputFile;

